src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorSingleStream.java

```diff
@@ -35,7 +35,6 @@
 public class RemoteBitrateEstimatorSingleStream
     extends SinglePacketTransformerAdapter
     implements CallStatsObserver,
-               RecurringRunnable,
```

Comment:
Why are we doing this (given that it's a single executor that run them all, so we're not saving threads)? Do you think this (executing only when packets arrive) will affect the performance of the RBE?